contracts/modules/cover/CoverNFTDescriptor.sol

```diff
@@ -0,0 +1,96 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.0;
+
+import "@openzeppelin/contracts-v4/utils/Strings.sol";
+import "@openzeppelin/contracts-v4/utils/Base64.sol";
+import "../../interfaces/ICover.sol";
+import "../../libraries/DateTime.sol";
+import "./CoverNFT.sol";
+
+contract CoverNFTDescriptor {
+  using Strings for uint256;
+  using DateTime for uint256;
+
+  string[] public coverAssets;
+
+  constructor() {
+    // TODO: how to handle asset symbols?
```

Comment:
Not sure what to do about displaying the symbols. Do we expect new cover assets often?

Reply:
Passing them via constructor would force us to deploy a new token descriptor every time the assets change. Should be pulled from the Pool contract instead. Probably could pass the master address via constructor and make it immutable.